packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/simulationservice/SimulationRequest.java

```diff
@@ -22,4 +23,7 @@ public class SimulationRequest implements Serializable {
 	private Object extra;
 
 	private String engine;
+
+	@TSOptional
+	private List<Intervention> interventions;
```

Comment:
Should this class be split into Julia and Ciemss? Im not sure if both take this parameter...